pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/component.go

```diff
@@ -53,7 +53,8 @@ type ComponentHandler struct {
 
 // Create implements EventHandler
 func (c *ComponentHandler) Create(evt event.CreateEvent, q workqueue.RateLimitingInterface) {
-	reqs, succeed := c.createControllerRevision(evt.Meta, evt.Object)
+	// TODO(zzxwill) evt.Meta in controller-runtime v0.6.2 is gone
+	reqs, succeed := c.createControllerRevision(evt.Object, evt.Object)
```

Comment:
@wonderflow @ryanzhang-oss In Kubernetes controller-runtime v0.8.0, event.CreateEvent doesn't have field `Meta`. I haven't found a way to fix it. Please help take look at it.